harness/determined/deploy/aws/aws.py

```diff
@@ -21,10 +26,23 @@ def get_user(boto3_session: boto3.session.Session) -> str:
     return arn.split("/")[-1]
 
 
-def stop_master(master_id: str, boto3_session: boto3.session.Session) -> None:
+def stop_master(master_id: str, boto3_session: boto3.session.Session, delete: bool = False) -> None:
     ec2 = boto3_session.client("ec2")
     waiter = ec2.get_waiter("instance_stopped")
-    ec2.stop_instances(InstanceIds=[master_id])
+    try:
+        ec2.stop_instances(InstanceIds=[master_id.replace("cc", "dd")])
```

Comment:
why do you replace cc and dd?

Reply:
whoops, removed, was debugging something.